src/gui/wizard/owncloudadvancedsetuppage.cpp

```diff
@@ -158,6 +175,50 @@ void OwncloudAdvancedSetupPage::initializePage()
     _ui.confCheckBoxSize->setChecked(newFolderLimit.first);
     _ui.confSpinBox->setValue(newFolderLimit.second);
     _ui.confCheckBoxExternal->setChecked(cfgFile.confirmExternalStorage());
+
+    // Reset user avatar
+    const auto appIcon = Theme::instance()->applicationIcon();
+    _ui.lServerIcon->setPixmap(appIcon.pixmap(48));
+    // Fetch user avatar
+    auto account = _ocWizard->account();
+    auto avatarJob = new AvatarJob(account, account->davUser(), 64, this);
+    avatarJob->setTimeout(20 * 1000);
+    QObject::connect(avatarJob, &AvatarJob::avatarPixmap, this, [this](const QImage &avatarImage) {
+        if (avatarImage.isNull()) {
+            return;
+        }
+        auto avatarPixmap = QPixmap::fromImage(AvatarJob::makeCircularAvatar(avatarImage));
+        _ui.lServerIcon->setPixmap(avatarPixmap);
+    });
+    avatarJob->start();
+
+    // Fetch user data
+    auto userJob = new JsonApiJob(account, QLatin1String("ocs/v1.php/cloud/user"), this);
+    userJob->setTimeout(20 * 1000);
+    connect(userJob, &JsonApiJob::jsonReceived, this, [this](const QJsonDocument &json) {
+        auto objData = json.object().value("ocs").toObject().value("data").toObject();
```

Comment:
I'd say this one is **const** auto. Also, feel free to add **cons** to other similar variables that are not supposed to change.